Wikipedia/Code/WMFCVLAttributes.h

```diff
@@ -0,0 +1,5 @@
+#import <UIKit/UIKit.h>
+
+@interface WMFCVLAttributes : UICollectionViewLayoutAttributes
```

Comment:
also it's shorter to write than UICollectionViewLayoutAttributes, but I understand if neither of these are compelling arguments